Source/PlasticSourceControl/Private/PlasticSourceControlMenu.cpp

```diff
@@ -629,6 +643,33 @@ void FPlasticSourceControlMenu::AddMenuExtension(FToolMenuSection& Menu)
 			FCanExecuteAction()
 		)
 	);
+
+	FPlasticSourceControlProvider& Provider = FPlasticSourceControlModule::Get().GetProvider();
+	const FString& serverUrl = Provider.GetServerUrl();
+
+	if (serverUrl.Contains("@cloud"))
+	{
+		const FString& organizationName = serverUrl.Left(serverUrl.Find("@cloud"));
+
+		Menu.AddMenuEntry(
+#if ENGINE_MAJOR_VERSION == 5
+			"PlasticLockRulesURL",
+#endif
+			LOCTEXT("PlasticLockRulesURL", "Configure Lock Rules"),
+			LOCTEXT("PlasticLockRulesURLTooltip", "Navigate to lock rules configuration page in the Unity Dashboard."),
+#if ENGINE_MAJOR_VERSION == 5 && ENGINE_MINOR_VERSION >= 1
+			FSlateIcon(FAppStyle::GetAppStyleSetName(), "PropertyWindow.Locked"),
+#elif ENGINE_MAJOR_VERSION == 5
+			FSlateIcon(FEditorStyle::GetStyleSetName(), "PropertyWindow.Locked"),
+#elif ENGINE_MAJOR_VERSION == 4
+			FSlateIcon(FEditorStyle::GetStyleSetName(), "PropertyWindow.Locked"),
```

Comment:
do you know if this icon was already available in UE4 and in UE5.1?
If yes we can simplify and collapse the 3 lines, if not we need to have a fallback icons like in other instances in the menu

Reply:
Hey @danielhompanera, good news I was able to find the "PropertyWindow.Locked" icon in the code source of UE4.27 as well :) 

so we can simplify the #ifdefs

Reply:
Oops, you went too far, this code doesn't compile on UE4 or UE5.0 since we need to keep the distinction between the old FEditorStyle::GetStyleSetName() and the new FAppStyle::GetAppStyleSetName()

I could have been more precise, sorry, see for instance

```
		LOCTEXT("PlasticProjectSettings",			"Project Settings - Source Control - Unity Version Control"),
		LOCTEXT("PlasticProjectSettingsTooltip",	"Open the Unity Version Control (formerly Plastic SCM) section in the Project Settings."),
#if ENGINE_MAJOR_VERSION == 5 && ENGINE_MINOR_VERSION >= 1
		FSlateIcon(FAppStyle::GetAppStyleSetName(), "ProjectSettings.TabIcon"),
#else
		FSlateIcon(FEditorStyle::GetStyleSetName(), "ProjectSettings.TabIcon"),
#endif
```